software/ADPL_electron/ADPL_electron.ino

```diff
@@ -200,9 +213,14 @@ int publish_data(int last_publish_time) {
             tempHXCI.temp, tempHXCO.temp, tempHTR.temp, tempHXHI.temp, tempHXHO.temp,
             int(valve.gasOn), int(bucket.tip_count));
 
-    publish_success = Particle.publish("DATA",data_str);
+    if (sdFile) {
+        sdFile.println(data_str);
```

Comment:
Not that I've found... but it could be undocumented.  The available documentation for a lot of these Arduino libraries is quite poor.  I was going to troll the src code a bit to confirm nothing is there.